### PR TITLE
[INLONG-10751][Agent] Report Agent process status for backend problem analysis

### LINKED ISSUE #10751: [Improve][Agent] Agent needs to report process status for backend problem analysis
### Description

Agent needs to report process status for backend problem analysis

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by justinwwhuang**
Fixes #10751 

### Motivation

 Report Agent process status for backend problem analysis

### Modifications

 Report Agent process status every minute, including version, os, cpu, memeory, sending static etc

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

No doc needed

### CODE REVIEW

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/store/InstanceStore.java [hunk 1]
```diff
@@ -54,6 +55,18 @@ public List<InstanceProfile> listAllInstances() {
         return instanceList;
     }
 
+    public int getRunningInstanceCount() {
+        List<KeyValueEntity> result = this.store.findAll(store.getUniqueKey());
+        int count = 0;
+        for (KeyValueEntity entity : result) {
+            if (entity.getAsInstanceProfile().getState() == InstanceStateEnum.DEFAULT) {
+                LOGGER.info("test123 {}", entity.getAsInstanceProfile().getInstanceId());
```
Comment (aloyszhang) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/store/InstanceStore.java:63]: please replace `test123` with some meaningful logs

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+        fields.add(String.valueOf(getProcessId()));
+        if (AgentManager.getAgentConfigInfo() != null) {
+            fields.add(AgentManager.getAgentConfigInfo().getMd5());
+        } else {
+            fields.add("");
+        }
+        fields.add(agentManager.getTaskManager().getTaskResultMd5());
+        fields.add(String.valueOf(agentManager.getTaskManager().getTaskStore().getTasks().size()));
+        fields.add(String.valueOf(OffsetManager.getInstance().getRunningInstanceCount()));
+        fields.add(ExcuteLinux.exeCmd("who -b|awk '{print $(NF-1), $NF}'").replaceAll("\r|\n", ""));
```
Comment (aloyszhang) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java:223]: There is no need to calculate the boot time for every report loop.

**Review by aloyszhang — APPROVED** [review]

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    public static final String INLONG_AGENT_STATUS = "inlong_agent_status";
+
+    private static AgentStatusManager manager = null;
+    private final AgentConfiguration conf;
+    private final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"); // 设置格式
+    private Runtime runtime = Runtime.getRuntime();
+    final long GB = 1024 * 1024 * 1024;
+    private OperatingSystemMXBean osMxBean;
+    private ThreadMXBean threadBean;
+    private long INVALID_CPU = -1;
```
Comment (doleyzi) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java:78]: Recommended use:
private final long INVALID_CPU = -1;

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+        return manager;
+    }
+
+    public void sendStatusMsg(List<String> fields) {
+        if (sender == null) {
+            LOGGER.error("sender is null");
+            createMessageSender();
+            return;
+        }
+        SendResult ret = sender.sendMessage(
```
Comment (doleyzi) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java:122]: The sender may have failed to be created previously, causing NPE. It is recommended to make a null judgment on the sender again.

Comment (justinwwhuang) [reply inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/AgentStatusManager.java:124]: there is an return

**Review by doleyzi — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**